CharacterCreation/01_Races.txt

```diff
@@ -120,87 +149,131 @@ Starting Stat Points: 40
+    Native to the once Ocean planet Trioch II, these beings evolved to the 
+changing of their planet as a massive solar flare from their star shifted the 
+magnetic fields holding in the atmosphere were shaken and ultimately disrupted, 
+causing the oceans to begin evaporating. With their oceans slowly dissipating 
+through the leak in the atmosphere, they have begun migrating to the nearby 
+Trioch IV, also mostly oceanus, which is out of immediate danger from the star. 
+Now, they live in both the oceans and the well-shaded estuaries on Trioch IV. A 
+few stubborn stragglers remain on Trioch II, which is slowly becoming a sandy, 
+dry wasteland with very little atmosphere. 
 
 
-Trioch II is now a place for venturing archaeologists to come and investigate the way the Opaleites used to live. 
+    Trioch II is now a place for venturing archaeologists to come and 
+investigate the way the Opaleites used to live. 
+
+===== Racial Bonuses =====
 
-Racial Bonuses:
 Waterbreathing
 +1 base FORT, +10 to reflex rolls. Due to their rapid evolution, Opaleites have 
 a much easier time adjusting to the physical obstacles presented to them. 
 -10 to perception rolls above water.
 Sensitivity to Smoke and Gasesous attacks. -20 to save rolls involving these 
 attacks. 
 
-        Subraces:
+===== Subraces =====
+
+== Seadevil ==
 
-Seadevil
 -1 to base CHA, -1 to base FORT.
 +1 to DEX
 Low-light vision.
 Ability: Bioluminescence - Once per day.[Is an available skill] Persuade a 
 being, sentient or not, that you are trustworthy. Victim must roll a will-save 
 50+(Siren points). Effect is dissipated when victims eyes are covered or lose 
 sight of you. This takes a tremendous amount of effort and you take a -10 
-penalty if tired or bloodied. This ability will temporarily scramble a robot’s 
+penalty if tired or bloodied. This ability will temporarily scramble a robot's 
 visual cortex, but they can adjust rapidly.
 
-Chondrich
+== Chondrich ==
+
 -1 to base CHA, +1 to both STR and FOR
 Tough Skinned: Natural Armor protects you in a firefight. +10 armor coverage, 
 + 5+(FOR/2) Damage reduction. Armor piercing rounds ignore these bonuses. 
 Large: You are larger than your counterparts and are thus easier to hit, -5 DC 
 on any chance to hit you.
 
-Mendar
+== Mendar ==
+
 +2 to base CHA
 -2 to chosen stats.
 Ability: Colorful! - People like you, they naturally flock toward your brilliant
 scales. You have the ability to either wow or calm a crowd. You are the perfect
 distraction; bring in the attention of those in the area. Victims must roll a 
 will-save of 40+Siren Points. Droids are immune to this effect.
 
-Tetra
+== Tetra ==
+
 -1 to DEX, -1 to STR

```

Comment:
The copyright notice at the bottom needs to get removed

Reply:
removed